src/current/_includes/releases/v24.2/v24.2.0.md

```diff
@@ -0,0 +1,144 @@
+- `storage.value_blocks.enabled`: The `storage.value_blocks.enabled` [cluster setting]({% link v24.2/cluster-settings.md %}) has been removed; value blocks are always enabled. [#122164][#122164]
+
+<h5 id="v24-2-0-settings-with-changed-defaults">Settings with changed defaults</h5>
+
+- `kv.dist_sender.circuit_breakers.mode` has had its default changed to `liveness range only = 1`.
+- `sql.defaults.results_buffer.size` has had its default changed to `512 KiB`: The default value of the `sql.defaults.results_buffer.size` [cluster setting]({% link v24.2/cluster-settings.md %}) has been changed from 16KiB to 512KiB. This reduces the chance that clients using [`READ COMMITTED`]({% link v24.2/read-committed.md %}) transactions will encounter errors that cannot automatically be retried within CockroachDB. [#124633][#124633]
+- `sql.metrics.max_mem_stmt_fingerprints`, `sql.metrics.max_mem_txn_fingerprints` have had default changed to `7500`: The default values for the [cluster settings]({% link v24.2/cluster-settings.md %}) `sql.metrics.max_mem_stmt_fingerprints` and `sql.metrics.max_mem_txn_fingerprints` have been changed from `100000` to `5000`, thus lowering the default limits for in-memory statement and transaction fingerprints. [#123430][#123430]
+- `sql.stats.histogram_samples.count` has had its default changed to `0`: Histograms are no longer constructed using a default sample size of `10k`. Samples are now dynamically sized based on table size unless the sample count has been set in the table or [cluster settings]({% link v24.2/cluster-settings.md %}). [#125345][#125345]
+- `sql.ttl.default_delete_rate_limit` has had its default changed to `100`: The storage parameter `ttl_delete_rate_limit`, which determines the rate limit for deleting expired rows, is now set to `100` by default. [#124354][#124354]
+
+<h5 id="v24-2-0-settings-with-new-options">Settings with new options</h5>
+
+- `storage.sstable.compression_algorithm` has added the (non-default) option `none = 3`: The compression option `none` was added to allow for the disabling of SSTable compression. This option can be used with any of the three existing cluster settings that control SSTable compression: `storage.sstable.compression_algorithm`, `storage.sstable.compression_algorithm_backup_storage`, and `storage.sstable.compression_algorithm_backup_transport`. [#126508][#126508]
+
+<h5 id="v24-2-0-settings-with-new-aliases">Settings with new aliases</h5>
+
+- `changefeed.batch_reduction_retry.enabled` is aliased to `changefeed.batch_reduction_retry_enabled`
```

Comment:
Are the new aliases preferred or we don't mind? This might be better formatted as a table.

Reply:
ya might be nice as a table next time

re: preferring aliases, idk - AFAICT we do not have clear product view in this area

historically cluster setting format / design has per-team ownership (and thus idiosyncratic format/design)